build-system/tasks/update-packages.js

```diff
@@ -23,6 +23,14 @@ const {exec, execOrDie, getStderr} = require('../exec');
 
 const yarnExecutable = 'npx yarn';
 
+// Transform files in node_modules since deps use ES6 export.
+// https://github.com/babel/babelify#why-arent-files-in-node_modules-being-transformed
+const es6Packages = [
```

Comment:
Is the idea this list will contain all modules containing `export` statements?

If so, would it make sense to more automatically discover these modules? I'm imagining a new `node_module` added in the future that misses this allowlist and fails to work as expected.

Reply:
This is a good concern. We could read the list of packages under `dependencies` in our root `package.json` and make sure all of them are transformed. WDYT?

Reply:
I like that a lot better. Good idea!